dbms/src/Flash/Mpp/MPPTunnel.cpp

```diff
@@ -392,4 +392,50 @@ void SyncTunnelSender::startSendThread(PacketWriter * writer)
         sendJob(writer);
     });
 }
+
+// TODO remove it in the future
+void MPPTunnel::connectUnrefinedLocal(PacketWriter * writer)
+{
+    {
+        std::unique_lock lk(mu);
+        if (status != TunnelStatus::Unconnected)
+            throw Exception(fmt::format("MPPTunnel has connected or finished: {}", statusToString()));
+
+        LOG_TRACE(log, "ready to connect");
+
+        RUNTIME_ASSERT(writer == nullptr, log);
+        local_tunnel_sender_v1 = std::make_shared<LocalTunnelSenderV1>(queue_size, mem_tracker, log, tunnel_id, &data_size_in_queue);
+        tunnel_sender = local_tunnel_sender_v1;
+
+        status = TunnelStatus::Connected;
+        cv_for_status_changed.notify_all();
+    }
+    LOG_DEBUG(log, "connected");
```

Comment:
Add information about the local tunnel version.

Reply:
> Add information about the local tunnel version.

okk